### DISCUSSION TITLE
Error when importing to abaqus

### DISCUSSION THREAD

**Discussion by sdantu2** [neperfepx/neper#873] [Q&A] [answered]
When I convert my mesh into a .inp file for importing into abaqus, I get the error of "in keyword *ELSET, file "assignedCrystalSizes.inp", line 93364: The following sets specified do not exist. Sets should be defined before they are used:TESS.29813," I am unsure of the reason why as I did not receive any errors during tessellation or meshing in Neper.

**Comment by Srijan7Y** [comment, marked as answer]
Try adding "_**-faset none**_" command in meshing line command while converting to .inp file, then try opening it in Abaqus.

**Reply by sdantu2** [reply]
Thank you very much! That fixed my issue and I was able to import into Abaqus without any issues!

**Reply by Anmol-Bakshi** [reply]
Can you share the command by which your error was resolved.

**Reply by Srijan7Y** [reply]
neper -T -n 10 -o try 
neper -M try.tess -format inp -faset none 

You will get the desired input file.

**Reply by Adiljan623** [reply]
Thank for the command, I tried to it and it worked, however, it generate orphan mesh, there was time in earlier version of Neper, I was be able to fully functional Neper generated Abaqus input file and fully worked as I wanted. I tried the same command, but it does not work in the same way before. do you have any idea  ?thanks

**Reply by Srijan7Y** [reply]
I have no idea about it. I generally, create .inp file with any domain shape of my tessellation from neper, and then I use it to do FEM or CPFEM in Abaqus without any issues.

**Reply by Anmol-Bakshi** [reply]
@Srijan7Y 
By using this command "neper -T -n 10 -o try
neper -M try.tess -format inp -faset none", in abaqus, are different grains present as different sets?
